src/queries/get_entities_from_sitelinks.ts

```diff
@@ -63,4 +65,4 @@ export function getEntitiesFromSitelinksFactory (buildUrl: BuildUrlFunction) {
 }
 
 // convert 2 letters language code to Wikipedia sitelinks code
-const parseSite = site => site.length === 2 ? `${site}wiki` : site
+const parseSite = (site: string) => (site.length === 2 ? `${site}wiki` : site)
```

Comment:
This is another problem I noticed but its not the scope of this PQ.

```suggestion
// TODO: sites are not always exactly length 2
const parseSite = (site: string) => (site.length === 2 ? `${site}wiki` : site)
```

Reply:
true, yeah it's unrelated to this PR.

Reply:
I would add this as a comment (for now) so its not forgotten. Or open an issue about it?